tutorials/java/chapters/ch07.adoc

```diff
@@ -0,0 +1,221 @@
+:imagesdir: images
```

Comment:
Yeah, perhaps it could be added to an appendix for now so that users can find out more information on Java EE management if they wish to learn more.  This way users can progress directly to the swarm/compose sections.  WDYT?